src/analysis/special_functions/integrals.lean

```diff
@@ -341,7 +341,7 @@ begin
   linarith,
 end
 
-lemma integral_sin_pow_antimono : ∫ x in 0..π, sin x ^ (n + 1) ≤ ∫ x in 0..π, sin x ^ n :=
+lemma integral_sin_pow_antitone : ∫ x in 0..π, sin x ^ (n + 1) ≤ ∫ x in 0..π, sin x ^ n :=
```

Comment:
I think that this should be called `integral_sin_pow_succ_le` and we should have `integral_sin_pow_antitone : antitone (λ n, ∫ x in 0..π, sin x ^ n)`.